static/js/components/tiles/disaster_event_map_tile.tsx

```diff
@@ -90,8 +95,9 @@ export function DisasterEventMapTile(
   const svgContainerRef = useRef(null);
   const infoCardRef = useRef(null);
   const europeanPlaces = useRef([]);
+  const dateRanges = useRef(getDateRanges());
   const [dateList, setDateList] = useState([]);
-  const [selectedDate, setSelectedDate] = useState("");
+  const [selectedDate, setSelectedDate] = useState(DATE_OPTION_30D_KEY);
```

Comment:
Maybe we should use DATE_OPTION_6M_KEY as default for now? There's no data for the last 30 days currently.

Reply:
good idea, updated